File: IntegracaoPHP/transfer.php
<?php
require("vendor/autoload.php");
$pagarme = new PagarMe\Client('ak_test_jbIXhrHXHOaUUNKtUVrkT9HGL60SSg');

$transfer = $pagarme->transfers()->create([
    'amount' => 1000,
    'recipient_id' => 're_ck0zjmw1s007ier6eysyd3agj'
]);
echo("A transferência: ".$transfer->id.", de ".$transfer->amount." centavos foi realizada para a conta bancária do recebedor ".$transfer->source_id."!");

File: IntegracaoPython/transfer.py
import pagarme
import json

def Transfer():

    pagarme.authentication_key('ak_test_jbIXhrHXHOaUUNKtUVrkT9HGL60SSg')

    transfer_params = {
        'amount': '1000',
        'recipient_id': 're_ck0zjmw1s007ier6eysyd3agj'
    }

    transfer = pagarme.transfer.create(transfer_params)
    transf_in_string = json.dumps(transfer)
    parsed_json = (json.loads(transf_in_string))

    return parsed_json["id"], parsed_json["amount"], parsed_json["source_id"]
